ci-operator/step-registry/cucushift/e2e/cucushift-e2e-commands.sh

```diff
@@ -130,9 +130,9 @@ function filter_test_by_proxy() {
     echo_e2e_tags
 }
 function filter_test_by_hypershift() {
-    local masterno
-    masterno="$(oc get nodes --no-headers -l node-role.kubernetes.io/master= | wc -l)"
-    if [[ $masterno -eq 0 ]] ; then
+    local topo
+    topo="$(oc get infrastructures.config.openshift.io cluster -o yaml | yq '.status.controlPlaneTopology')"
+    if [[ -n "$topo" ]] && [[ "$topo" = 'External' ]] ; then
```

Comment:
Do we really need 2 criteria here? I think [[ "$topo" = 'External' ]] is enough

Reply:
Done